config/environments/production.rb

```diff
@@ -60,12 +60,29 @@
   # Use a real queuing backend for Active Job (and separate queues per environment)
   # config.active_job.queue_adapter     = :resque
   # config.active_job.queue_name_prefix = "magma-beers_#{Rails.env}"
+  # amazon config
+  config.action_mailer.default_url_options = { host: 'magma-beers.herokuapp.com' }
+  config.action_mailer.perform_deliveries = true
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.smtp_settings = {
+    address: ENV['SES_SMTP_SERVER'],
+    port: 587,
+    user_name: ENV['SES_SMTP_USERNAME'],
+    password: ENV['SES_SMTP_PASSWORD'],
```

Comment:
Remember to use `ENV.fetch("variable")` instead of only `ENV` to raise an error when we haven't set some environment variable.